packages/ag-charts-enterprise/src/features/annotations/annotationUtils.ts

```diff
@@ -100,7 +100,7 @@ export function invertCoords(coords: Coords, context: AnnotationContext) {
     return { x, y };
 }
 
-export function invert(n: Coords['x' | 'y'], context: AnnotationAxisContext) {
+export function invert(n: Coords['x' | 'y'], context: AnnotationAxisContext | _ModuleSupport.AxisContext) {
     const halfBandwidth = (context.scaleBandwidth() ?? 0) / 2;
```

Comment:
Perhaps narrow down the type to `Pick<AnnotationAxisContext, 'scaleBandwidth' | 'scaleInvert' | 'scaleInvertNearest'>`? Not sure, but since these two classes overlap so much, it is a little odd to have both here?